transport_plugins/bled112/iotile_transport_bled112/bled112.py

```diff
@@ -69,6 +69,8 @@ def __init__(self, port, on_scan=None, on_disconnect=None, passive=None, **kwarg
             self._active_scan = config.get('bled112:active-scan')
 
         self._throttle_broadcast = config.get('bled112:throttle-broadcast')
+        self._throttle_scans = config.get('bled112:throttle-scan')
+        self._throttle_scans_timeout = config.get('bled112:throttle-scan-timeout')
```

Comment:
Since this applies to both scan and broadcasts it should just be called `bled112:throttle-timeout`

Reply:
Yes, it looks better